app/src/main/java/com/example/fond/adapters/RecipeSearchAdapter.java

```diff
@@ -131,8 +133,9 @@ public void onClick(View view) {
         public void bind(Recipe recipe) {
             boolean isSaved = false;
 
-            tvRecipeTitle.setText(recipe.getTitle());
```

Comment:
I think this line got inadvertently deleted because the titles for each recipe in the search screen was missing (even though you could still tap them to go to the recipe details screen).